templates/web/base/waste/garden/subscribe_details_bin_desc.html

```diff
@@ -0,0 +1,38 @@
+<div class="waste-service-hint-banner">
+  <div class="waste-service-hint-banner__image">
+    [% IF cobrand.moniker == 'brent' %]
+      <img src="/i/waste-containers/bin-green.png" alt="">
+    [% ELSIF cobrand.moniker == 'bromley' %]
+      <img src="/i/waste-containers/bin-black-brown-lid-recycling.png" alt="">
+    [% ELSIF cobrand.moniker == 'kingston' %]
+      <img src="/i/waste-containers/bin-grey-green-lid-recycling.png" alt="">
+    [% ELSIF cobrand.moniker == 'merton' %]
+      [% image = PROCESS "waste/garden/wheelie.svg" %]
+      <span style="--primary-color: #8B5E3D;">[% image.replace('<svg', '<svg style="height:auto;max-width:100%;"') | safe %]</span>
+    [% ELSIF cobrand.moniker == 'sutton' %]
+      <img src="/i/waste-containers/bin-green-brown-lid.png" alt="">
+    [% ELSE %]
+    [% END %]
+  </div>
+  <div class="waste-service-hint__content">
+    <p>
+      [% IF cobrand.moniker == 'brent' %]
+        Our 240 litre wheelie bins are green, and hold the equivalent of 5 or 6 bags of green garden waste.
+      [% ELSIF cobrand.moniker == 'bromley' %]
+        Each wheelie bin holds the equivalent of 5 or 6 bags of green garden waste,
+        and will be collected fortnightly during gardening season (March–November)
+        and monthly outside of gardening season (December–February).
+      [% ELSIF cobrand.moniker == 'kingston' %]
+        Each garden waste bin holds 240 litres of waste. We collect them every 2 weeks.
+      [% ELSIF cobrand.moniker == 'merton' %]
+        Our garden waste bins are brown. They have a 140L or 240L
+        capacity, which is about the same size as a small or standard wheelie bin.
+      [% ELSIF cobrand.moniker == 'sutton' %]
+        Each garden waste bin holds 240 litres of waste. We collect them every 2 weeks.
+      [% ELSE %]
+        Each garden waste bin holds 240 litres of waste. We collect them every 2 weeks.
+      [% END %]
```

Comment:
Could this if statement be tidied up by calling a method in the cobrand modules? Or even pull from general.yml? Same for the banner image above (though possibly trickier with Merton's special case).

(And same question for the subscribe_existing_ template below)

Reply:
I guess the only downside of config is that it means you have to set that up in order for the cobrand to work okay, makes it a bit harder to have the equivalent of what's being shown, iyswim - and for something like this is very unlikely to change. I think I prefer output text in the templates rather than the code in general. As this commit was doing, it was in separate per-cobrand template files, which was cleaner (though every one had to repeat the little boilerplate bits) but also harder to see that each one was a little bit customised only for a line or two of text - I thought this way was overall to see that each cobrand has its own tiny custom output.